contracts/Overtime/SportsAMMV2.sol

```diff
@@ -757,11 +773,39 @@ contract SportsAMMV2 is Initializable, ProxyOwned, ProxyPausable, ProxyReentranc
         );
     }
 
-    function _exerciseTicket(address _ticket) internal {
+    function _exerciseTicket(address _ticket, address _exerciseCollateral, bool _inEth) internal {
         Ticket ticket = Ticket(_ticket);
-        ticket.exercise();
-        IERC20 ticketCollateral = ticket.collateral();
+        (uint userWinningAmount, address ticketOwner, IERC20 ticketCollateral) = ticket.exercise(_exerciseCollateral);
```

Comment:
ticketOwner and ticketCollateral can be called directly from Ticket

Reply:
saves gas by adding 3 external calls into a single call